src/model/documentselection.js

```diff
@@ -359,6 +359,14 @@ export default class DocumentSelection {
 		return this._selection.hasAttribute( key );
 	}
 
+	/**
+	 * Refreshes selection attributes and markers according to the current position in the model.
+	 */
+	refreshAttributes() {
```

Comment:
This should be named `refresh()`.

Reply:
Or... different. It doesn't look good when `refreshAttributes` refreshes more than attributes. Or simply let's not wrap it in a function.

Reply:
`refresh()` sounds ok. I was considering it.